tests/integration/Db/Adapter/Pdo/Mysql/ListTablesCest.php

```diff
@@ -86,8 +86,9 @@ public function dbAdapterPdoMysqlListTables(IntegrationTester $I)
             'stock',
             'subscriptores',
             'table_with_string_field',
+            'tiny_int_test',
             'tipo_documento',
-            'users',
```

Comment:
@ruudboon https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
Good catch. Thnx